notebooks/Analysis_with_dataset_GNPS_15_12_2021/data_processing_and_training_models/9_create_training_data_ms2query.py

```diff
@@ -3,27 +3,27 @@
 
 file_date = "15_12_2021"
 
-path_data = "C:\\HSD\\OneDrive - Hochschule Düsseldorf\\Data\\ms2query"
+path_data = "C:/Users/jonge094/PycharmProjects/PhD_MS2Query/ms2query/data/libraries_and_models/gnps_15_12_2021/"
 
 assert os.path.exists(os.path.join(path_data, "library_GNPS_15_12", f"library_GNPS_{file_date}.sqlite"))
```

Comment:
```suggestion
path_library = os.path.join(path_data, "library_GNPS_15_12")

assert os.path.exists(os.path.join(path_library, f"library_GNPS_{file_date}.sqlite"))
```

Reply:
If you agree with this change, you can also click on "Add suggestion to batch" or "commit suggestion".